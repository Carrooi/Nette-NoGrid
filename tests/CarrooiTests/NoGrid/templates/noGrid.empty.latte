<table n:no-grid="grid">
	<tr n:no-grid-data-as="$line">
		<td>{$line[number]}</td>
	</tr>
	<tr n:no-grid-empty>
		<td>Empty</td>
	</tr>
</table>
