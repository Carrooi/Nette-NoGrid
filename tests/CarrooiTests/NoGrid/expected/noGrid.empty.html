<table>
	<tr>
		<td>Empty</td>
	</tr>
</table>
